lib/galaxy/config/__init__.py

```diff
@@ -34,6 +34,7 @@
 
 import yaml
 
+from galaxy.carbon_emissions import get_carbon_intensity_entry
```

Comment:
This is breaking the config module

Reply:
Is this still causing an issue and, if so, how exactly is it breaking the config module?

Reply:
it is, the config package can't import from carbon_emissions.

Reply:
Work on a fix is underway in #17159

Reply:
Thanks for the update!